OSSM-8862 Migration Hub page content (#88651)
openshift/openshift-docs: modules/ossm-migrating-hub-completing-your-migration.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/ossm-migrating-from-service-mesh-2-to-3-assembly.adoc

:_mod-docs-content-type: REFERENCE
[id="ossm-migrating-hub-completing-your-migration_{context}"]
= Completing your migration

> Depending on your deployment model, there may be extra steps to follow to complete the migration. For example, there are extra steps for migrating with the cert-manager tool that must be completed before you can remove {SMProduct} 2 resources.

Reviewer: ```suggestion
Depending on your deployment model, you might need to take extra steps to complete the migration. For example, when you migrate with the cert-manager tool, you need to take additional steps before you can remove {SMProduct} 2 resources.
```

> After you have completed your migration, you can add new workloads, recreate network policies, and remove {SMProduct} 2 resources.

Reviewer: ```suggestion
After you complete your migration, you can add new workloads, re-create network policies, and remove {SMProduct} 2 resources.
```
